.env

```diff
@@ -0,0 +1 @@
+API_BASE_URL = 'http://localhost:3000'
```

Comment:
I don't think we should commit .env file.. We usually use .env.development file.